src/app/pages/vm/vm-wizard/vm-wizard.component.ts

```diff
@@ -70,7 +70,7 @@ export class VmWizardComponent implements WizardConfiguration {
   private isolatedGpuPciIds: string[];
 
   entityWizard: EntityWizardComponent;
-  private productType = window.localStorage.getItem('product_type') as ProductType;
+  private productType = this.systemGeneralService.getProductType();
```

Comment:
let's refactor this class properties? We have private and public 
Let's sort them by, **public** and **private** - currently they are mixed